src/frontend/components/UI/TextInputField/index.tsx

```diff
@@ -64,6 +64,7 @@ const TextInputField = ({
         )}
         rightSection={inputIcon}
         label={label}
+        readOnly={true}
```

Comment:
can we expose the props and use `readOnly={true}` where we need it instead of setting it for all implementations?

Reply:
🫡

Reply:
made the change. great suggestion